test/credit_card_sanitizer_test.rb

```diff
@@ -102,6 +102,7 @@ class CreditCardSanitizerTest < MiniTest::Test
         assert_nil @sanitizer.sanitize!("blah blah  +4111111111111111.json")
         assert_nil @sanitizer.sanitize!("\"+4111111111111111\"")
         assert_nil @sanitizer.sanitize!("(+4111111111111111)")
+        assert_nil @sanitizer.sanitize!("&#43;1 936 321 1111")
```

Comment:
The reason this one doesn't get redacted is because the recognized number is `434111111111111111` which I guess does not pass Luhn check.

```
irb(main):008:0> CreditCardSanitizer.new.sanitize!("&#43;4111111111111111")
=> nil
```

This one, since you changed the `1` to a `2`, probably doesn't pass Luhn check either:

```
irb(main):025:0> CreditCardSanitizer.new.sanitize!("&#43;2 936 321 1111")
=> nil
```